packages/website/plugins/generated-rule-docs.ts

```diff
@@ -50,9 +50,11 @@ export const generatedRuleDocs: Plugin = () => {
       return;
     }
 
-    const rule = pluginRules[file.stem];
+    const rule = pluginRules[file.stem] as
+      | (typeof pluginRules)[keyof typeof pluginRules]
+      | undefined;
     const meta = rule?.meta;
-    if (!meta?.docs) {
+    if (!rule || !meta?.docs) {
```

Comment:
Nit: unnecessary change?

```suggestion
    if (!meta?.docs) {
```